packages/sdk-core/package.json

```diff
@@ -84,18 +83,17 @@
         "eslint-plugin-eslint-comments": "^3.2.0",
         "eslint-plugin-import": "^2.25.4",
         "eslint-plugin-prettier": "^3.3.1",
-        "ethereum-waffle": "^3.4.0",
+        "ethereum-waffle": "^3.4.4",
         "get-graphql-schema": "^2.1.2",
-        "hardhat": "^2.8.3",
-        "mocha": "^9.2.0",
+        "hardhat": "^2.9.1",
+        "mocha": "^9.2.2",
         "npm-run-all": "^4.1.5",
         "nyc": "^15.1.0",
-        "rollup": "^2.67.0",
-        "ts-node": "^10.4.0",
+        "ts-node": "^10.7.0",
         "typechain": "^6.1.0",
         "typedoc": "^0.22.11",
-        "typescript": "^4.5.5",
-        "web3": "^1.7.0"
+        "typescript": "^4.6.2",
+        "web3": "^1.7.1"
     },
```

Comment:
I'm not sure it makes sense to update these common packages separately in this project?

Reply:
this is going to be addressed in the https://github.com/superfluid-finance/protocol-monorepo/pull/694